act-rules-format.bs

```diff
@@ -127,6 +131,7 @@ An ACT Rule SHOULD list an identifier and reference for each accessibility requi
 
 One reason why an [=atomic rule=] might not list accessibility requirements is because it is part of a [=composite rule=], and only a `failed` [=outcome=] in the composite rule would mean not satisfying the accessibility requirement. Another reason might be that an ACT Rule is designed to evaluate a best practice that is not part of any accessibility standard.
 
+This section MAY be left blank.
```

Comment:
I think this is redundant or I don't understand it. We have a SHOULD requirement already, so by definition it MAY be empty. Maybe you want to say that this heading MUST be in the rule document but that it MAY be empty?